src/specification/models/taxon/index.json

```diff
@@ -2,117 +2,117 @@
   "additionalProperties": false,
   "properties": {
     "parent": {
-      "type": "object", 
+      "type": "object",
       "properties": {
         "id": {
-          "type": "string", 
+          "type": "string",
           "description": "Parent taxon identifier"
-        }, 
+        },
         "scientific_name": {
-          "type": "string", 
+          "type": "string",
           "description": "Parent taxon scientific name"
-        }, 
+        },
         "rank": {
-          "type": "string", 
+          "type": "string",
           "description": "Parent taxon rank"
         }
-      }, 
+      },
       "required": [
-        "id", 
-        "scientific_name", 
+        "id",
+        "scientific_name",
         "rank"
       ]
-    }, 
-    "rubin_number": {
```

Comment:
Should this really be changed to plural?

Reply:
Great catch!